zappa/zappa.py

```diff
@@ -326,9 +334,19 @@ def create_lambda_zip(self, prefix='lambda_package', handler_file=None,
                 venv = bin_path[:bin_path.rfind(env_name)] + env_name
                 logger.debug('env path = {}'.format(venv))
             else:  # pragma: no cover
-                print("Zappa requires an active virtual environment.")
+                print("Zappa requires an active virtual or conda environment.")
                 quit()
 
+            if conda_mode:
```

Comment:
It changes the way packaging works. In the standard virtualenv mode, zappa takes the `site-packages` folder of the virtualenv and possibly replaces some binaries using `lambda-packages`. In `conda_mode` zappa does a few things:

0. Create a temporary copy of the conda environment for the following manipulations 
1. Remove excluded conda packages (like python itself)
2. Move the `site-packages` folder to the root of the conda environment 
3. Zip up this moved `site-packages` along with all other remaining binaries in the conda environment.